types/sizzle/index.d.ts

```diff
@@ -3,6 +3,13 @@ export as namespace Sizzle;
 declare const Sizzle: SizzleStatic;
 export = Sizzle;
 
+// For users who don't have "dom" types
+/* eslint-disable @typescript-eslint/no-empty-interface */
+interface Document {}
+interface DocumentFragment {}
+interface Element {}
```

Comment:
This doesn't work because this file is a module, so they will never be declared with any props, and therefore are like typing everything with `any`. If you meant to polyfill these, they have to be in a `global` block or something.

Reply:
_OH_ I didn't understand the first comment. Thanks, got it!